### ISSUE TITLE
xgettext fails to parse JS files containing template strings

### ISSUE THREAD

**Issue by Ortham** [loot/loot#1065]
The pre-commit hook that runs xgettext to generate template.pot prints:

```
src/gui/html/elements/loot-custom-icons.js:32: warning: RegExp literal terminated too early
src/gui/html/elements/loot-groups-editor.js:151: warning: unterminated string
src/gui/html/elements/loot-menu.js:38: warning: RegExp literal terminated too early
src/gui/html/elements/loot-plugin-card.js:209: warning: unterminated string
src/gui/html/js/updateExists.js:77: warning: unterminated string
```

This must have been happening for a while, but I didn't notice because I rarely run `git commit` directly. xgettext doesn't support JS template strings, there's a [bug open](https://savannah.gnu.org/bugs/?50920) about it, but it might never be fixed, so it's worth looking for alternatives to xgettext or even gettext for the JS files.

**Comment by Ortham** [comment]
A quick search around didn't reveal any xgettext reimplementations that support JS template strings, but [i18next](https://www.i18next.com/) looks like it could work as a partial replacement to gettext - it's got its own format, but looks like it has two-way interop with gettext, and also plays well with ICU's MessageFormat, so using it wouldn't be very limiting.

**Comment by Ortham** [comment]
Fixed in 0ca360a5638d599579973db6f93b5b399a6f409c using babel-gettext-extractor. It's a little messy, but it works.
